tests/integration/src/test/java/org/mule/test/core/context/notification/FlowStackTestCase.java

```diff
@@ -64,8 +62,8 @@ public void subFlowStatic() throws Exception {
     assertThat(stackToAssert, not(nullValue()));
 
     assertStackElements(stackToAssert,
-                        isFlowStackElement("subFlow", "/subFlowStatic/processors/0/subFlow/subprocessors/0"),
-                        isFlowStackElement("subFlowStatic", "/subFlowStatic/processors/0"));
+                        isFlowStackElement("subFlow", "subFlow/processors/0"),
```

Comment:
Yes, in fact it used to be a problem. For context data about previous execution they have the flow stack.